python/GafferImageUI/ImageViewUI.py

```diff
@@ -164,14 +113,25 @@ def _updateFromPlugs( self ) :
 
 		],
 
+		"displayTransformGPU" : [
+			"description",
+			"""
+			Applies a gamma correction to the image.
+			""",
+
+			"plugValueWidget:type", "GafferImageUI.ImageViewUI._LutGPUPlugValueWidget",
+			"toolbarLayout:index", 5,
```

Comment:
Might it make more sense to put this after the displayTransform dropdown rather than before? So it reads "apply this exposure/gamma/clipping/transform using this method"?

Reply:
Addressed by Tom.